docs/main/user-guides/migration/migration-using-pv-migrate.md

```diff
@@ -0,0 +1,303 @@
+ - Local PV Device to Local PV LVM
+ - Local PV Device to Local PV ZFS
+ - Migration from OpenEBS cStor to OpenEBS Replicated
+ - cStor to Replicated
+ - cStor to Mayastor
+ - Jiva to Replicated
+ - Jiva to Mayastor
+description: This section outlines the process of migrating the legacy storage to latest storage solution.
+---
+
+# Migration using pv-migrate
+
+This section describes the process of migrating the legacy storage to latest storage solution.
+
+## Overview
+
+OpenEBS data migration is the process of transferring data from the legacy storage to the latest storage solution. The data can be migrated from any non-OpenEBS storage to OpenEBS storage and this section provides detailed description about the migration process for legacy storage users.
+
+The data migration of migrating the legacy storage to latest storage solution can be done in two ways:
+- [Migration using pv-migrate](#migration-using-pv-migrate)
+- [Migration using velero](../migration/migration-using-velero/)
+
+In this migration process, we are using [pv-migrate](https://github.com/utkuozdemir/pv-migrate) that is a CLI tool/kubectl plugin to easily migrate the contents of one Kubernetes `PersistentVolumeClaim` to another.
+
+This tool is binary and can be [downloaded](https://github.com/utkuozdemir/pv-migrate/releases/download/v1.7.1/pv-migrate_v1.7.1_linux_x86_64.tar.gz) from the release section for linux/amd64. For other OS and arch, download the respective binary from the latest [release section](https://github.com/utkuozdemir/pv-migrate/releases/tag/v1.7.1).
+
+1. Once downloaded, untar the binary as below:
+
+```
+tar -xvf pv-migrate_v1.7.1_linux_x86_64.tar.gz
+```
+
+2. Add the binary to `PATH` or move it to `/usr/local/bin` to use the binary like any usual binary.
+
+```
+mv pv-migrate /usr/local/bin
+```
+
+The binary can be used as specified in the migrate flows.
+
+## Migration from Local PV Device to Local PV LVM
+
+:::info
+The following steps are an example about migrating from legacy storage to latest storage solution.
+You can also migrate OpenEBS Local PV Device to OpenEBS Local PV ZFS.
+:::
+
+### Assumptions
+
+- Local PV Device is already deployed.
+- MongoDB Standalone is deployed as below using the Local PV Device PVC. (Here, MongoDB Standalone is an example.)
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: localpv-vol
+spec:
+  storageClassName: openebs-device
+  accessModes: ["ReadWriteOnce"]
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+- For validation, some data has been inserted in the MongoDB as an example below:
+
+```
+db.admin.insertMany([{name: "Max"}, {name:"Alex"}])
+
+[
+  { _id: ObjectId('65eaafa01cd2b6de45285d86'), name: 'Max' },
+  { _id: ObjectId('65eaafa01cd2b6de45285d87'), name: 'Alex' }
+]
+```
+### Steps to migrate Local PV Device to Local PV LVM
+
+Follow the steps below to migrate OpenEBS Local PV Device to OpenEBS Local PV LVM.
```

Comment:
to migrate data from OpenEBS Local PV Device storage to OpenEBS Local PV LVM storage